api/repositories/org_repository_test.go

```diff
@@ -471,32 +471,37 @@ var _ = Describe("OrgRepository", func() {
 		})
 	})
 
-	Describe("PatchOrgMetadata", func() {
+	Describe("PatchOrg", func() {
 		var (
 			orgGUID                       string
+			displayName                   string
+			orgNewName                    *string
 			cfOrg                         *korifiv1alpha1.CFOrg
 			patchErr                      error
 			orgRecord                     repositories.OrgRecord
 			labelsPatch, annotationsPatch map[string]*string
 		)
 
 		BeforeEach(func() {
-			cfOrg = createOrgWithCleanup(ctx, prefixedGUID("org-name"))
+			displayName = prefixedGUID("org-name")
+			cfOrg = createOrgWithCleanup(ctx, displayName)
 			orgGUID = cfOrg.Name
 			labelsPatch = nil
 			annotationsPatch = nil
+			orgNewName = tools.PtrTo("new-org-name")
```

Comment:
I wonder whether using a hardcoded new name could cause name collisions when running tests in parallel. Would recommend `uuid.NewString()` instead of a hardcoded value